third_party/pcsc-lite/naclport/server_clients_management/src/client-handler.js

```diff
@@ -301,11 +302,6 @@ ClientHandler.prototype.serverRequesterDisposedListener_ = function() {
   if (this.isDisposed())
```

Comment:
nit: This check is not needed anymore as well.

Reply:
I don't understand this. Why?

Reply:
We can just call `this.dispose()`. Same as in the other comment thread, it can be called multiple times, but it does internal bookkeeping in order to call `disposeInternal()` only once.

Reply:
ah, thanks, i was looking at the wrong part of the code